homeassistant/components/sensor/onewire.py

```diff
@@ -18,10 +18,19 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+REQUIREMENTS = ['pyownet==0.10.0']
+
 CONF_MOUNT_DIR = 'mount_dir'
 CONF_NAMES = 'names'
+CONF_SERVER_HOST = 'server'
+CONF_SERVER_PORT = 'port'
+CONF_SERVER_TIMEOUT = 'timeout'
```

Comment:
Please use `CONF_HOST`, `CONF_PORT`, etc. as those are already defined in `const.py`.